create-react-app/src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
```

Comment:
I think we're hoping to do propagate a pattern like `*.cy.(test|spec).js` to help disambiguate between existing Jest tests

Reply:
That makes sense, so people who might be adding Cypress tests into an already-tested project will be able to do so incrementally. Will something need to be done on the Jest side of things to prevent it from looking at Cypress tests?

Ideally, we'd recommend a file naming pattern that doesn't require changing any of the existing Jest config!

Reply:
I talked to @amirrustam about this, we're going to use `.spec.ct.{js,ts,jsx,tsx}`